packages/system/src/components/Formatters/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './Currency';
```

Comment:
Can we also have a `Number` formatter :D (Maybe we need to name it `Numeric` because "Number" is part of the language)

Reply:
yes sure 🙂